paper-server/patches/sources/net/minecraft/server/commands/GiveCommand.java.patch

```diff
@@ -8,7 +8,31 @@
          int maxStackSize = itemStack.getMaxStackSize();
          int i = maxStackSize * 100;
          if (count > i) {
-@@ -95,11 +_,11 @@
+@@ -65,8 +_,10 @@
+                     i1 -= min;
+                     ItemStack itemStack1 = item.createItemStack(min, false);
+                     boolean flag = serverPlayer.getInventory().add(itemStack1);
++
++                    // Paper start - Fix PlayerDropItemEvent dupe on /give
++                    ItemEntity itemEntity = serverPlayer.drop(itemStack1, false, false, false, null);
```

Comment:
this broke how that works you replace two drop calls one with itemStack and another with itemStack1 for a method where only itemStack1 is used.

better modify the current calls for add the other params and include the // Paper

but im unsure if the fix is avoid the event or improvement the fallback what sounds a little hard because the method dont know from where come the drop...